spectree/plugins/starlette_plugin.py

```diff
@@ -38,9 +39,27 @@ def register_route(self, app):
             )
 
     async def request_validation(self, request, query, json, form_data, headers, cookies):
+        data = {}
+        if request.headers.get('content-type', '').split(";")[0] in self.FORM_MIMETYPE:
```

Comment:
if get the default `''` and split here index[0]  will raise an error

Reply:
Thanks for the feedback!
You're right, splitting an empty string with a `None` separator returns [] and `''.split()[0]` will raise an error.
But in this case `split()` uses a separator `;`. So `''.split(';')[0]` will return an empty string
From Python documentation:
`Splitting an empty string with a specified separator returns [''].`
Refs: https://docs.python.org/3/library/stdtypes.html#str.split

Reply:
got your point, you are right, cool~